### PR TITLE
Updated documentation for External License Master, External indexer c…

### PR DESCRIPTION
**Pull request by akondur**
- Updated documentation for **External License Master.**
- Updated documentation for **External Indexer Cluster**.
- Updated documentation for changing Splunk Docker container image version **8.0.x** to **8.1.0**. The Splunk Docker container image version in **operator.yaml** and **splunk.v0.2.0.clusterserviceversion.yaml** will be done via the following [task](https://jira.splunk.com/browse/CSPL-506) post Beta release, once rarity is released and Splunk Docker Image version is available on Docker Hub.
- Copyright file modified to meet legal requirements.

### CODE REVIEW

File: docs/Examples.md [hunk 1]
```diff
@@ -620,50 +620,56 @@ spec:
...
-...
-[general]
-pass4SymmKey = $7$Sw0A+wvJdTztMcA2Ge7u435XmpTzPqyaq49kUZqn0yfAgwFpwrArM2JjWJ3mUyf/FyHAnCZkE/U=
-...
-```
+There are two ways to configure `pass4Symmkey` with an External License Master:
 
-You can decrypt the `pass4SymmKey` by running the following command with
-`--value` set to the value from of your `server.conf` file:
+#### Approach 1
```
[the remark this answers is not in the document]
Comment (akondur) [reply docs/Examples.md:634]: In this approach, there are two cases:

1. You could already have an LM configured with a pass4Symmkey but you would just re-configure the pass4Symmkey to the desired plain-text pass4Symmkey.

2. LM is not configured with a pass4Symmkey and you would just configure the pass4Symmkey to the desired plain-text pass4Symmkey for the first time.

File: docs/Examples.md [hunk 2]
```diff
@@ -620,50 +620,56 @@ spec:
...
+- Setup the desired plain-text [`pass4Symmkey`](PasswordManagement.md#pass4symmkey) in the global secret object(Note: The `pass4Symmkey` would be stored in a base64 encoded format). For details see [updating global kubernetes secret object](#updating-global-kubernetes-secret-object).
+- Setup the same plain-text `pass4SymmKey` in the `[general]` section of your license master's `server.conf` file.
 
-```
-$SPLUNK_HOME/bin/splunk show-decrypted --value '$7$Sw0A+wvJdTztMcA2Ge7u435XmpTzPqyaq49kUZqn0yfAgwFpwrArM2JjWJ3mUyf/FyHAnCZkE/U='
-```
+#### Approach 2
```
[the remark this answers is not in the document]
Comment (akondur) [reply docs/Examples.md:638]: When a LM is up and running, Splunk makes sure that there is a pass4symmkey configured for sure? Is there a need for us to consider a case where there is no pass4symmkey configured?

File: docs/Examples.md [hunk 3]
```diff
@@ -620,50 +620,56 @@ spec:
...
-you may create a `default.yml` file with the following contents:
+  You can decrypt the `pass4SymmKey` by running the following command with
+  `--value` set to the value from your `server.conf` file:
+
+  ```
+  $SPLUNK_HOME/bin/splunk show-decrypted --value '$7$Sw0A+wvJdTztMcA2Ge7u435XmpTzPqyaq49kUZqn0yfAgwFpwrArM2JjWJ3mUyf/FyHAnCZkE/U='
+  ```
+- Setup the above decrypted plain-text [`pass4Symmkey`](PasswordManagement.md#pass4symmkey) in the global secret object(Note: The `pass4Symmkey` would be stored in a base64 encoded format). For details see [updating global kubernetes secret object](#updating-global-kubernetes-secret-object)
+
+### Configuring license_master_url:
```
[the remark this answers is not in the document]
Comment (akondur) [reply docs/Examples.md:656]: As discussed offline, if the networking to the cluster is good the services configured will make sure that CR's in the cluster are reachable.

**Review by gaurav-splunk — APPROVED** [review]

**Review by eddikay — APPROVED** [review]

File: docs/Examples.md [hunk 4]
```diff
@@ -620,50 +620,56 @@ spec:
 
 ## Using an External License Master
 
-*Note that this requires using the Splunk Enterprise container version 8.0.3 or later*
+*Note that this requires using the Splunk Enterprise container version 8.1.0 or later*
 
 The Splunk Operator for Kubernetes allows you to use an external license
```
[the remark this answers is not in the document]
Comment (akondur) [reply docs/Examples.md]: Done

File: docs/ChangeLog.md [hunk 5]
```diff
@@ -1,6 +1,7 @@
 # Splunk Operator for Kubernetes Change Log
 
 ## 0.2.0 Beta (2020-10-15)
+* This release depends upon changes made concurrently in the Splunk Enterprise container images. You must use the latest splunk/splunk:edge nightly image with it, or alternatively any release version 8.1.0 or later.
```
Comment (sgontla) [inline docs/ChangeLog.md:4]: Nit: Comment used in other places makes more formal. i.e "*Note that this requires using the Splunk Enterprise container version 8.1.0 or later*"

**Review by smohan-splunk — APPROVED** [review]

File: docs/ChangeLog.md (hunk 5 above)
Comment (akondur) [reply docs/ChangeLog.md:4]: @sgontla I was trying to maintain consistency with Changelog.md entry for 0.1.0 alpha [here](https://github.com/splunk/splunk-operator/blob/develop/docs/ChangeLog.md#010-alpha-2020-03-20).

Comment (sgontla) [reply docs/ChangeLog.md:4]: We may need to avoid suggesting the nightly build, if not now, later on, a full release...

**Review by sgontla — APPROVED** [review]

**Pull request merged**